crates/epaint/Cargo.toml

```diff
@@ -49,6 +49,13 @@ deadlock_detection = ["dep:backtrace"]
 ## If you plan on specifying your own fonts you may disable this feature.
 default_fonts = ["epaint_default_fonts"]
 
+## Disable text snapping to remove logic in text layout that rounds glyphs and kerning to
+## the nearest pixel. This is necessary to provide the sharpest text rendering with `ab_glyph`.
+## The side effect of such rounding is that text layout will likely be inconsistent across
+## different values of `pixel_per_point`. Disabling such "snapping" will make the text fuzzy,
+## but the layout will remain constant across `pixels_per_point`.
+disable_text_snapping = []
```

Comment:
That makes sense. Let me revise it.

Reply:
To me, it makes the most sense to add this as a field of `LayoutJob` which would default to `true` for snapping. That would allow a user like myself to have, e.g., the UI elements all snap while retaining more granular control over a particular layout job. I'm not sure if adding a Struct field is considered a breaking change, so I'd welcome your thoughts on that. Thanks for a great library @emilk.

Reply:
I think adding it to `LayoutJob` is the right thing yes